srcpkgs/limine/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'limine'
+pkgname=limine
+version=3.0.3
+revision=1
+archs="x86_64 x86_64-musl"
```

Comment:
`x86_64*`, but it really only works on x86_64? What issues are there for e.g. `i686*`? It's marketed as a `x86/x86_64 BIOS/UEFI bootloader` after all in the `short_desc`

Reply:
Works on i686, does not build on i686, Ι also have a WIP PR: https://github.com/void-linux/void-packages/pull/36380

Reply:
> This builds and works on i686.

Oh, I see, you use the toolchain for cross-building. Nice :) Are you going to update the PR to the most recent version. I will close mine :)

Reply:
I just updated the PR to version 3.4.4, and thanks for the interest in the project!